vcd_cli/network.py

```diff
@@ -765,6 +770,38 @@ def modify_ip_range_external_network(ctx, name, gateway_ip, ip_range,
         stderr(e, ctx)
 
 
+@external.command(
+    'remove-ip-range',
+    short_help='Removes an IP range of a subnet in an external network.')
+@click.pass_context
+@click.argument('name', metavar='<name>', required=True)
+@click.option(
+    '-g',
+    '--gateway-ip',
+    'gateway_ip',
+    required=True,
+    metavar='<ip>',
+    help='gateway ip of the subnet')
+@click.option(
+    '-i',
+    '--ip-range',
+    'ip_range',
+    required=True,
+    metavar='<ip>',
+    help='ip range in StartAddress-EndAddress format')
+def remove_ip_range_external_network(ctx, name, gateway_ip, ip_range):
```

Comment:
I feel We should provide user do delete multiple ip ranges, as from ui we can do it the samw and even in gateway we are doing the same?, please provide your input

Reply:
I just checked even confluence page says the same for removing ip range:

https://confluence.eng.vmware.com/pages/viewpage.action?spaceKey=VCD&title=Design+of+External+Network+commands+for+vcd-cli